pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -22,15 +22,15 @@ import (
 
 // ClientConnectionConfiguration contains details for constructing a client.
 type ClientConnectionConfiguration struct {
-	// kubeConfigFile is the path to a kubeconfig file.
-	KubeConfigFile string `json:"kubeconfig"`
+	// kubeconfig is the path to a KubeConfig file.
+	Kubeconfig string `json:"kubeconfig"`
```

Comment:
I guess we are not worried about backward compatibility issues here, right?

Reply:
actually, the json name is the same. So, NVM.

Reply:
No, we aren't. However, the JSON is backwards-compatible with this change.
This is all alpha, and we're trying to get to a state where we can move types out, do a full field-by-field review on everything, change stuff if needed and then graduate individual components' types to `v1beta1` and higher.

Reply:
Sounds good. I already approved. Thanks!